src/Control/FunFlow/Exec/Redis.hs

```diff
@@ -1,13 +1,14 @@
-{-# LANGUAGE Arrows, GADTs, OverloadedStrings, TupleSections,
-       TypeFamilies, GeneralizedNewtypeDeriving #-}
```

Comment:
Should aim to follow the Tweag style guide: https://github.com/tweag/guides/blob/master/style/Haskell.md
In this case, one extension per line, rather than multiple extensions in a single pragma. `stylish-haskell` will do this automatically for you.